### ISSUE TITLE
Remove dependency on external HTTP URLs (e.g. vestacp.com)

### ISSUE THREAD

**Issue by n1trux** [n1trux/hestia#2]
The current problem is that you can't install vesta without having a connection to vestacp.com, which is bad.

An optimal solution would probably be a DEB/RPM package for every distribution which includes every file necessary for installing vesta. This might be a tedious task to do without automation.

Any other suggestions?

**Comment by peterbrinck** [comment]
One of the great things about Vesta, is that it's so easy and fast to install, you don't have to download anything, extract anything and so on, just run a command and Vesta will take care of the rest for you.

I think we should think of a solution that keeps that simplicity.

I have no experience in creating and/or maintaining a DEB/RPM package, so I don't know how it should be handled, but it would certainly be the best and simplest solution for users.

**Comment by n1trux** [comment]
The thing with DEB/RPM packages for me would be that we could leverage already existing infrastructure and possibly also include the software in upstream repositories. That's way safer and more comfortable than downloading vesta from a possibly untrusted third party page.

Imagine just doing "dnf install vestacp" and having the panel run? :o

**Comment by peterbrinck** [comment]
You've just answered your own question then :wink: 
I think we should go that way, and have a DEB/RPM package. I don't know about this, but I guess there's some automated stuff to create the packages from GitHub, right?
So we would only have to create a new release, and the packages would be updated automatically?

**Comment by n1trux** [comment]
Yep, for that the code would need a major clean-up, though. I'm currently looking at the installer scripts and thinking about how to clean them up; you'll find me in #bash @ irc.freenode.net in the meanwhile :'D

When we resolved the installer issue, we can go on with creating packages. I'll open a new Issue for that though.

**Comment by n1trux** [comment]
The HTTP resources for vesta can be downloaded like this:

    wget 'http://c.vestacp.com/debian' -m
    wget 'http://c.vestacp.com/ubuntu' -m
    wget 'http://c.vestacp.com/rhel' -m

I'll be uploading these to a separate folder in the root directory of this repository and make the installers use these instead of the online resources. The current size of all the downloaded resources on my disk are approx. 30MiB.

**Comment by peterbrinck** [comment]
That's not too bad. That won't hurt anything
